### ISSUE TITLE
Image Fit, Seek Bar & Video Playback

### ISSUE THREAD

**Issue by xd1986k** [morkt/GARbro#120]
Hello
I have a few requests

1. An option to fit image within preview window.

2. A seek bar for audio files.

3. Video Playback support.

**Comment by morkt** [comment]
1. maybe
2. unlikely
3. no

**Comment by 0xh3LL0** [comment]
seconded for the 2nd feature, if its not possible, how about just add play/pause button?
since garbro can play audio inside archive, but now there is no way to stop the audio playback unless we close/exit garbro, so i think this feature will be useful.

**Comment by morkt** [comment]
yeah, stop button seems necessary, any other sound controls not so much. the way i see it is that, for example, you're looking for track you liked in game, extract it, and then play it in your regular media player.

**Comment by xd1986k** [comment]
Then, would it be possible to add a "preview in explorer" option for a file (which would extract to temp and immediately open it)? The current extract option takes too many steps to view a file in explorer.

**Comment by shamashii** [comment]
About 1,
Is it possible copy the preview image to clipboard(raw material) ? Then one can directly paste it to other apps when seek some image temporary. Just a low priority request.
